pkgs/development/tools/rome/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, stdenv
+, Security
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rome";
+  version = "11.0.0";
+
+  src = fetchFromGitHub {
+    owner = "rome";
+    repo = "tools";
+    rev = "cli/v${version}";
+    sha256 = "sha256-pQz1hs4oYPPQ8zqkwstDH3VefaYQXJ9uCkeC+0IMeSs=";
+  };
+
+  cargoSha256 = "sha256-9WTjb8pgk/qyIU8mPT6RF2FZptNylvILHMYvZDyRSMs=";
+
+  cargoBuildFlags = [ "--package" "rome_cli"];
+
+  preBuild = ''
+    export RUSTFLAGS="-C strip=symbols"
+    export ROME_VERSION=${version}
+  '';
+
+  # need to manually unset the ROME_VERSION before checkPhase otherwise some tests fail
+  postBuild = ''
+    unset ROME_VERSION;
+  '';
+
+  buildInputs = lib.optional stdenv.isDarwin [ Security ];
+
+  # these test fail
+  checkFlags = [
+    "--skip commands::format::does_not_format_ignored_directories"
+    "--skip parser::tests::uncompleted_markers_panic"
+  ];
+
+  # takes long time on x86_64-darwin
+  requiredSystemFeatures = [ "big-parallel" ];
+
+  meta = with lib; {
+    description = "A formatter, linter, bundler, and more for JavaScript, TypeScript, JSON, HTML, Markdown, and CSS";
+    homepage = "https://rome.tools";
+    license = licenses.mit;
+    maintainers = with maintainers; [ dit7ya ];
```

Comment:
Feel free to add me as a maintainer. I'd love to make sure `rome` is up-to-date in nixpkgs.